app/controllers/articles_controller.rb

```diff
@@ -49,7 +49,7 @@ def destroy
   private
     # Use callbacks to share common setup or constraints between actions.
     def set_article
-      @article = Article.find(params[:id])
+      @article = Article.friendly.find(params[:id])
```

Comment:
let's user `find_by_slug!` here instead of URLs are not accessible with `/articles/1` as well. https://github.com/norman/friendly_id/issues/146